Description: Embded libcache.h into libndpi includes, as it is included by ndpi_typedefs.h
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ndpi/src/include/ndpi_typedefs.h
===================================================================
--- ndpi.orig/src/include/ndpi_typedefs.h
+++ ndpi/src/include/ndpi_typedefs.h
@@ -25,7 +25,7 @@
 #define __NDPI_TYPEDEFS_H__
 
 #include "ndpi_define.h"
-#include "../lib/third_party/include/libcache.h"
+#include "libcache.h"
 
 #define BT_ANNOUNCE
 #define SNAP_EXT
Index: ndpi/src/lib/Makefile.am
===================================================================
--- ndpi.orig/src/lib/Makefile.am
+++ ndpi/src/lib/Makefile.am
@@ -12,7 +12,8 @@ libndpi_la_include_HEADERS = ../include/
 			     ../include/ndpi_includes.h \
 			     ../include/ndpi_protocol_ids.h \
 			     ../include/ndpi_protocols.h \
-			     ../include/ndpi_typedefs.h
+			     ../include/ndpi_typedefs.h \
+			     ../include/libcache.h
 
 libndpi_la_SOURCES = ndpi_content_match.c.inc \
 		     ndpi_main.c \
